src/plugins/explore/public/application/legacy/discover/application/components/visualizations/explore_visualization.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiPanel } from '@elastic/eui';
+import React from 'react';
+import { IExpressionLoaderParams } from '../../../../../../../../expressions/public';
+import { VisualizationTypeResult } from './utils/use_visualization_types';
+import { LineChartStyleControls } from './line/line_vis_config';
+
+export interface ExploreVisualizationProps {
+  expression: string;
+  searchContext: IExpressionLoaderParams['searchContext'];
+  styleOptions: LineChartStyleControls;
+  visualizationData: VisualizationTypeResult;
+  onStyleChange: (newOptions: Partial<LineChartStyleControls>) => void;
+  ReactExpressionRenderer: React.ComponentType<{
+    expression: string;
+    searchContext: IExpressionLoaderParams['searchContext'];
```

Comment:
search Context here vs the search context at the root

Reply:
what do you mean by this? @kavilla